Utest/rimea_tests/reportscript.py

```diff
@@ -245,13 +258,17 @@ def get_git_status():
 		:return: branch, author, date, commit of the newst commit(tuple)
 		"""
 		## Get the branch name by `git status` command
-		branch = subprocess.check_output(["git", "status"]).splitlines()[0].split(' ')[-1]
+		branch =str(subprocess.check_output(["git", "status"]))
```

Comment:
This code doesn't work on windows. Is there any way to solve it?

Reply:
Why it does not work?
Can you post the error?

Reply:
> Why it does not work?
> Can you post the error?

I think we can not call git on windows in this way
![image](https://user-images.githubusercontent.com/32732507/62128926-d0a89c80-b2d5-11e9-8f51-388f9996f6f2.png)

Reply:
@xuqiancheng Any suggestions?

Reply:
IIRC the windows installer for git does not add git to the PATH variable automatically. When you start the git shell the environment gets populated with the correct path. For any other program to find git on windows the path to git has to be added to PATH

See:https://stackoverflow.com/questions/26620312/installing-git-in-path-with-github-client-for-windows

Reply:
Maybe you can try this hack first: 

```
import subprocess

proc = subprocess.Popen('git.cmd status')
print ('result: ', proc.communicate())
```

https://stackoverflow.com/questions/4965175/make-subprocess-find-git-executable-on-windows

Reply:
> Maybe you can try this hack first:
> 
> ```
> import subprocess
> 
> proc = subprocess.Popen('git.cmd status')
> print ('result: ', proc.communicate())
> ```
> 
> https://stackoverflow.com/questions/4965175/make-subprocess-find-git-executable-on-windows

Ugh, so the solution would be to search for both `git.exe` or `git.cmd` on windows and then use the respective call :/

Reply:
@gjaeger I  have tried several ways from internet, but didn't work.

@Ozaq Thanks, but I have already added git to the PATH variable, and I can use git in cmd
![image](https://user-images.githubusercontent.com/32732507/62132097-b32b0100-b2dc-11e9-9121-640971cd3220.png)

@chraibi I have tried several ways including this one, but didn't work.
![image](https://user-images.githubusercontent.com/32732507/62132266-0309c800-b2dd-11e9-8efb-d54000d224c4.png)

Reply:
@Ozaq 
You are right, I find that I have made some mistakes with the path of git and pycharm. 
After adjusting, it works good now. 
It's my fault, the code works on windows.

Reply:
> @Ozaq
> You are right, I find that I have made some mistakes with the path of git and pycharm.
> After adjusting, it works good now.
> It's my fault, the code works on windows.

Nice! No extra code for windows! Yay!